Doxyfile

```diff
@@ -0,0 +1,2 @@
+INPUT = include/ src/
+OUTPUT_DIRECTORY       = ./docs/
```

Comment:
Is this executed? Where are the generated file stored, GitHub or?

Reply:
No it is not executed yet. Just a file we can use if we chose to